html-aam/index.html

```diff
@@ -4697,7 +4697,10 @@ <h4 id="el-legend">`legend`</h4>
               </th>
               <td>
                 <div class="role"><span class="type">Roles:</span> `ROLE_SYSTEM_STATICTEXT`; `IA2_ROLE_LABEL`</div>
-                <div class="relations"><span class="type">Relations:</span> `IA2_RELATION_LABEL_FOR` with the parent <a href="#el-fieldset">`fieldset`</a></div>
+                <div class="relations">
+                  <span class="type">Relations:</span> `IA2_RELATION_LABEL_FOR` with the parent <a href="#el-fieldset">`fieldset`</a> 
+                  or <a href="#el-optgroup">`optgroup`</a>
```

Comment:
For clarity, I think we need to specify the reverse LABELLED_BY relation somewhere. For example, [we do this for fieldset](https://w3c.github.io/html-aam/#:~:text=Relations%3A%20IA2%5FRELATION%5FLABELLED%5FBY%20with%20the%20first%20instance%20of%20a%20rendered%20child%20legend%20element). We're somewhat inconsistent about where we document the relation. For IA2, we document LABELLED_BY in the `<fieldset>` section, but for UIA, we document LabeledBy in the `<legend>` section. That's an editorial issue we can sort out separately, but I think it does need to be documented somewhere, and I'd suggest documenting the IA2 relation in the `<optgroup>` section, since that's what we do for `<fieldset>`.